### PR TITLE
Document adhocracy4 release process

### PR DESCRIPTION
**Pull request by slomo**

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Performing releases on A4
+=========================
+
+Basic properties of our release cycle:
+
+ - version numbers follow semantic versioning
+ - current development version is on branch `master`
+ - releases can only be on release branches named `releas<major>.<minor>`
+ - patch releases are tags on the minor branch
```
Comment (xi) [inline docs/releases.md]: lists do not start with whitespace in markdown

File: docs/releases.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Performing releases on A4
+=========================
+
+Basic properties of our release cycle:
+
+- version numbers follow semantic versioning
+- current development version is on branch `master`
+- releases can only be on release branches named `releas<major>.<minor>`
+- patch releases are tags on the minor branch
```
Comment (xi) [inline docs/releases.md]: what is a minor branch?

File: docs/releases.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+How to perform a release
+------------------------
+
+    OLD_VERSION=0.1.0
+    NEW_VRSION=0.2.0
```
Comment (xi) [inline docs/releases.md]: these variables are not used

**Comment by xi** [comment]
I understand everything you are saying here, but I do not get the big picture:

- What are the specific requirements to versioning in a4?
- When is a release branch created?
- How are hotfixes done? (how are they included in master and all relevant release branches?)

File: docs/releases.md (hunk 3 above)
Comment (slomo) [inline docs/releases.md]: I know, this is documentation ;)

File: docs/releases.md (hunk 2 above)
Comment (slomo) [inline docs/releases.md]: A branch for a specific minor version.

**Comment by slomo** [comment]
I could see including those as open questions, but I simply have no anwser for the first two questions.

 For the last one I would propose to only maintain the latest release for now. And fix all issues on that release branch, while merging it into master from time to time to get the fixes as well.

**Pull request merged**